src/client/actions/VoiceStateUpdate.js

```diff
@@ -11,7 +11,8 @@ class VoiceStateUpdate extends Action {
     if (guild) {
       // Update the state
       const oldState =
-        guild.voiceStates.cache.get(data.user_id)?._clone() ?? new VoiceState(guild, { user_id: data.user_id });
+        guild.voiceStates.cache.get(data.user_id)?._clone() ??
+        new VoiceState(guild, { user_id: data.user_id, suppress: data.suppress });
```

Comment:
why don't we just spread the object as a whole? will that cause issues?

Reply:
That will cause both voice states to be identical, that doesn't sound ideal.
I think it'd be better to default `suppress` to `true` as the PR initially did.

Reply:
I see your point! The other option I was thinking of was making it `null`, just in case `VoiceState` is swept by the `makeCache` `ClientOption`, since we can't tell what that was? I'm not familiar with the new cache stuff, but if it's no longer in cache and removed whilst they're a speaker in a stage channel, defaulting to `true` if they leave wouldn't be correct here? If I'm totally wrong, let me know and I'll change the default back to `true` (:

Reply:
I have just tested this out with this:
```JavaScript
const { Client, Options } = require("discord.js");

const client = new Client({
  intents: [
    "GUILD_VOICE_STATES"
  ],
  makeCache: Options.cacheWithLimits({
    VoiceStateManager: 3
  })
});

client.on("voiceStateUpdate", oldState => console.log(oldState)); // Let >= 3 people do whatever first, then leave as a speaker

client.login("token");
```
With a couple of accounts (3), their voice states were created accordingly. However, as a speaker of the stage channel, leaving still rendered `suppress` as `undefined`, so I believe defaulting to `null` would be more appropriate than `true` here. Thoughts?

Reply:
Personally I think `null` might be more adequate than defaulting to true... @SpaceEEC @discordjs/the-big-3?

Reply:
I'd like us emitting `null` as voice state if we have none in cache, but that's v14 area.

I think making `supress` nullable falls into that area too, but it wouldn't be as drastic.
So it's probably fine? 🤷 

I definitely prefer `null` over `true`, if that is possible.

Reply:
I think `null` is fine for the next minor version of discord.js (I see this pull request is aiming for version 13.1 release). If so, I'll go ahead and amend this! (: